README.md

```diff
@@ -51,29 +52,31 @@ The following example shows the format of the text file containing frame filenam
 ./anim1/frame04.png 600
 ```
 
-**WebP Encoding Options:**
+**Options:**
 
-| Option | Default Value | Description|
-|--------|:-------------:|------------|
+| <img width="150"> Option | Default Value | <img width="600"> Description|
```

Comment:
Is there another way to center the text rather than manual padding with `<img>` tags?

Reply:
I've tried several ways but it didn't work.

Reply:
Up to you but it feels like a trick. If the purpose was to fix the broken flags split on two lines, just remove the `-` I guess